library/pod-security-policy/capabilities/suite.yaml

```diff
@@ -0,0 +1,11 @@
+kind: Suite
+apiVersion: test.gatekeeper.sh/v1alpha1
+tests:
+  - name: capabilities
+    template: template.yaml
+    constraint: samples/capabilities-demo/constraint.yaml
+    cases:
```

Comment:
Add an allowed example (ditto others - it's important to verify that we document what is allowed)

Reply:
Done!